tensorflow_addons/layers/tests/octave_convolutional_test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for octave convolutional layers."""
+
+import numpy as np
+import pytest
+
+from tensorflow import keras
+from tensorflow_addons.layers.octave_convolutional import OctaveConv1D
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_octave_conv1d():
+    # verify output shape with padding
+    kwargs = {"filters": 3, "kernel_size": 3, "low_freq_ratio": 0.5, "padding": "same"}
+    layer = OctaveConv1D(**kwargs)
+    y = layer(keras.backend.variable(np.ones((1, 10, 2))))
```

Comment:
```suggestion
    y = layer(np.ones((1, 10, 2)))
```

We use eager mode in the tests. Numpy tensors are automatically converted to tf tensors.